### DISCUSSION TITLE
Initializing Solution in Newton Iterations

### DISCUSSION THREAD

**Discussion by nikhilgv91** [idaholab/moose#17384] [Q&A General] [answered]
Hello,

I am solving a transient problem. I would like to know whether it is possible to initialize the solution before the Netwon iterations for each timestep begin?

Best,
Nikhil

**Comment by dschwen** [comment, marked as answer]
You can look into the `Predictor` system. We have objects in that system that extrapolate past solutions to obtain a better guess. However you'll be dealing with raw DOF value access.
